src/colmap/exe/model.cc

```diff
@@ -44,27 +44,26 @@
 namespace colmap {
 namespace {
 
-std::vector<std::pair<Eigen::Vector3d, Eigen::Vector3d>>
-ComputeEqualPartsBounds(const Reconstruction& reconstruction,
-                        const Eigen::Vector3i& split) {
-  std::vector<std::pair<Eigen::Vector3d, Eigen::Vector3d>> bounds;
-  const auto bbox = reconstruction.ComputeBoundingBox();
-  const Eigen::Vector3d extent = bbox.second - bbox.first;
+std::vector<Eigen::AlignedBox3d> ComputeEqualPartsBboxes(
```

Comment:
A lot of code in this file should be extracted into its own module with some tests. Left for future work.